src/components/NumberChart.jsx

```diff
@@ -77,22 +77,22 @@ export default class NumberCharts extends React.Component {
         const lowValueColor = config.lowValueColor || currentTheme.number.style.labels.lowValue.fill;
 
         return (
-            <svg height="100%" width="100%" viewBox={`0 0 ${width} ${height}`}>
+            <svg height="100%" width="100%" viewBox={`0 0 800 450`}>
```

Comment:
Why hardcode `width` & `height`? Are not taking them as parameters?

Reply:
ok, understood. Since the `viewBox` is `0 0 800 450` max-width=800px & max-height=450px, why this limitation is in place?

BTW: IMO, if user passes improper width & height and that messes-up the chart, then that is on the user. User is responsible to pass proper dimensions to display the chart appropriately.

Reply:
Fixed to scale to title length